src/actions/deleteThought.ts

```diff
@@ -53,6 +55,9 @@ const deleteThought = (state: State, { local = true, pathParent, thoughtId, orph
 
   // See: Payload.local
   const persist = local || remote
+  if (Capacitor.isNativePlatform()) {
+    Haptics.notification({ type: NotificationType.Warning })
+  }
```

Comment:
One suggestion might be creating our own utility for haptics which checks for `Capacitor.isNativePlatform()` internally so that we don't need to include so many new `if` blocks throughout the rest of the code.

e.g.:

```suggestion
  haptics.notification({ type: NotificationType.Warning })
```

A reasonable counterargument is that we want to expressly show within the hooks/actions/components that the haptic event is only triggered on native.

I would be okay with leaving this as it is, but slightly prefer creating a utility.

Reply:
We definitely want to abstract this out since the condition is present with every Haptics call. It should always NOOP on other platforms.

Reply:
Refactored in the new PR here: a8ab9aec4919b3983d394df433b52a57b9e3402d